io.audio.MasVisGtk.json

```diff
@@ -0,0 +1,389 @@
+{
+    "id" : "io.audio.MasVisGtk",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "masvisgtk",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home:create"
```

Comment:
```suggestion
        "--filesystem=home"
```

Reply:
Would that not stop saving of analysis files?

Reply:
what does "create home directory" means to you?

Reply:
Exactly. It will be fixed.